<!DOCTYPE html> <html> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Compare ratios of different proteins in the same sample | Ginny Xiaohe Li </title> <meta name="author" content="Ginny Xiaohe Li"> <meta name="description" content="not what we normally do with TMT data"> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://ginnyintifa.github.io/blog/2023/distill/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> <script src="/assets/js/distillpub/template.v2.js"></script> <script src="/assets/js/distillpub/transforms.v2.js"></script> <script src="/assets/js/distillpub/overrides.js"></script> </head> <body> <d-front-matter> <script async type="text/json">
      {
            "title": "Compare ratios of different proteins in the same sample",
            "description": "not what we normally do with TMT data",
            "published": "May 22, 2023",
            "authors": [
              
            ],
            "katex": {
              "delimiters": [
                {
                  "left": "$",
                  "right": "$",
                  "display": false
                },
                {
                  "left": "$$",
                  "right": "$$",
                  "display": true
                }
              ]
            }
          }
    </script> </d-front-matter> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Ginny Xiaohe</span> Li </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/blog/">blog </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">events </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="post distill"> <d-title> <h1>Compare ratios of different proteins in the same sample</h1> <p>not what we normally do with TMT data</p> </d-title> <d-article> <d-contents> <nav class="l-text figcaption"> <h3>Contents</h3> <div> <a href="#tmt-background">TMT background</a> </div> <div> <a href="#compare-ratios-horizontally-between-samples-within-a-protein">Compare ratios horizontally between samples within a protein</a> </div> <div> <a href="#compare-ratios-vertically-between-proteins-within-a-sample">Compare ratios vertically between proteins within a sample</a> </div> <div> <a href="#a-real-example">A real example</a> </div> </nav> </d-contents> <h2 id="tmt-background">TMT background</h2> <p>TMT is short for tandem mass tag. When you deal with proteomics data with large sample size (N &gt;= 20), you are most likely dealing with TMT data. TMT is a very smart way to achieve efficient quantification of proteins. In a TMT10-plex experiment, 10 channels (or samples) are being quantified simultaneously, so that technical artefact is minimized (compared to doing 10 times separately).</p> <p>Usually, you would have multiple plexes. Say you have 50 samples of interest, you may have more than 5 TMT-10 plexes in the experiment. How to control for the technial artefact between the 5 plexes? You may assign the last channel of each plex (or even more than 1 channel) to be a physical average of all the samples of interest. By doing so, you have a common reference in each of the plex, which to be leveraged in merging plexes together.</p> <p><a href="https://fragpipe.nesvilab.org/" rel="external nofollow noopener" target="_blank">FragPipe</a> , the best proteomics analysis platform, calculates a ratio between each channel to the reference channel, and these ratios then directly merged (think a <code class="language-plaintext highlighter-rouge">join</code>) across different plexes to generate a single data table, with each row being a gene/protein, and each column a sample.</p> <h2 id="compare-ratios-horizontally-between-samples-within-a-protein">Compare ratios horizontally between samples within a protein.</h2> <p>Now we are happy to see that within each gene (row), ratios can be safely compared between samples. You could conduct a differential expression analysis (DE) between groups of interest.</p> <h2 id="compare-ratios-vertically-between-proteins-within-a-sample">Compare ratios vertically between proteins within a sample.</h2> <p>But can I compare different genes within a single sample (column)? Would such analysis make any sense? I have to say, during my 3+ years experience of analysing TMT data, I never saw anyone directly performed such comparison, nor did anyone asked for such analysis. It just seems to be very weird!!! But let me tell you, <em>it makes perfect sense!</em></p> <p>Why? We subtracted the intensity of the bridge channel (physical average) from each channel, then the ratio is a relative measurement telling you how big or small this gene/protein is expressed for this sample, compared to an average level.</p> <p>In fact, the bridge sample serves as a mediator in eliminating many types of biases such as ionization efficiency and also accounts for the stochastic nature of MS/MS.</p> <p>Actually if we want to perform a ssGSEA (<a href="https://www.nature.com/articles/nature08460" rel="external nofollow noopener" target="_blank">original method</a>), we are going to use this ratio data.</p> <h2 id="a-real-example">A real example</h2> <p>I grabbed proteomics from a CPTAC lung squamous cell carcinoma (<a href="https://www.cell.com/cell/fulltext/S0092-8674(21)00857-6" rel="external nofollow noopener" target="_blank">LSCC)</a> project . In this dataset, we have both tumor samples and adjacent normal tissues (NAT). Supposedly, normal samples are more homogeneous to each other comparing to tumor samples (intertumoral heterogeneity). In the LSCC dataset, each plex was assigned with balanced tumor and normal sample channels. The bridge channels, i.e. a common reference, physical average of all samles, was used to adjust for between-plex difference (batch effect). We would expect the proteins expressed highly in one NAT sample relative to other proteins are also highly expressed in other NAT samples. To test it, I replace the actual ratios of protein for each sample with the ranks from high to low, I plot all the protein ranks against the rank of a randomly picked sample.</p> <p>It looks like this:</p> <p><img src="/assets/img/rank_1vsall.png" alt="rank"></p> <p>The interpretation would be: if we randomly rank the protein ratios of one sample from high to low, then the sum of ranks over all the samples roughly goes from high to low as well, indicating it is justifiable to compare ratios of different proteins in the same sample.</p> <p>I did a ssGSEA with the ratios and here is a heatmap visualizating the results:</p> <p><img src="/assets/img/ssGSEA.png" alt="ssGSEA"></p> <p>Ratios are “rankable” values across proteins. Notice that the left most patient was excluded in downstream analysis due to sample quality.</p> </d-article> <d-appendix> <d-footnote-list></d-footnote-list> <d-citation-list></d-citation-list> </d-appendix> <d-bibliography src="/assets/bibliography/"></d-bibliography> <div id="giscus_thread" style="max-width: 930px; margin: 0 auto;"> <script>
      let giscusTheme = determineComputedTheme();
      let giscusAttributes = {
        src: 'https://giscus.app/client.js',
        'data-repo': 'ginnyintifa/ginnyintifa.github.io',
        'data-repo-id': '',
        'data-category': 'Comments',
        'data-category-id': '',
        'data-mapping': 'title',
        'data-strict': '1',
        'data-reactions-enabled': '1',
        'data-emit-metadata': '0',
        'data-input-position': 'bottom',
        'data-theme': giscusTheme,
        'data-lang': 'en',
        crossorigin: 'anonymous',
        async: '',
      };

      let giscusScript = document.createElement('script');
      Object.entries(giscusAttributes).forEach(([key, value]) => giscusScript.setAttribute(key, value));
      document.getElementById('giscus_thread').appendChild(giscusScript);
    </script> <noscript>Please enable JavaScript to view the <a href="http://giscus.app/?ref_noscript" rel="external nofollow noopener" target="_blank">comments powered by giscus.</a> </noscript> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Ginny Xiaohe Li. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>